diffrax/_term.py

```diff
@@ -776,3 +785,135 @@ def _to_vjp(_y, _diff_args, _diff_term):
         dy, vjp = jax.vjp(_to_vjp, y, diff_args, diff_term)
         da_y, da_diff_args, da_diff_term = vjp((-(a_y**ω)).ω)
         return dy, da_y, da_diff_args, da_diff_term
+
+
+LangevinX = PyTree[Shaped[Array, "?*langevin"], "LangevinX"]
```

Comment:
It's not clear to me from the name what this quantity represents.

Reply:
I agree it is unclear, I added the following comment:

```
# The Underdamped Langevin SDE trajectory consists of two components: the position
# `x` and the velocity `v`. Both of these have the same shape. So, by LangevinX we
# denote the shape of the x component, and by LangevinTuple we denote the shape of
# the tuple (x, v).
```